### PR TITLE
Add header component and to the pages

### PR DESCRIPTION
**Pull request by BerePendzialek**

### CODE REVIEW

File: src/components/MusicPage/MusicPage.js [hunk 1]
```diff
@@ -9,53 +9,79 @@ import {
...
+          onCreatePlaylist={onCreatePlaylist}
+        />
+
+        <CardGrid>
+          <Title>
+            Your selected workout:
+            <br /> {name}
+          </Title>
+
+          <Wrapper>
```
Comment (Joao17Duarte) [inline src/components/MusicPage/MusicPage.js:35]: Wrapper for what?
 maybe it's ok, but it's easier if you have a specific nameWrapper

File: src/components/MusicPage/MusicPage.js [hunk 2]
```diff
@@ -9,53 +9,79 @@ import {
...
+    display: block;
+    height: 58px;
+    width: 100%;
+  }
+  &::after {
+    content: '';
+    display: block;
+    height: 58px;
+    width: 100%;
+  }
```
Comment (Joao17Duarte) [inline src/components/MusicPage/MusicPage.js:81]: why the before and after they have the same style?

Comment (BerePendzialek) [reply src/components/MusicPage/MusicPage.js:81]: this is more less my grid. whatever in between plus 2 sections over and under with the same height. I will consider this for the final layout.

File: src/components/MusicPage/MusicPage.js (hunk 1 above)
Comment (BerePendzialek) [reply src/components/MusicPage/MusicPage.js:35]: ok. I´ll be mor specific. Thanks!

**Pull request merged**